Vimeo-DotNet/VimeoAPI/packages.config

```diff
@@ -1,4 +1,4 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Newtonsoft.Json" version="5.0.5" targetFramework="net40" />
+  <package id="Newtonsoft.Json" version="6.0.8" targetFramework="net46" />
```

Comment:
We're trying to update `Newtonsoft.Json` to 7.0.1(It's in the Trello card template for reference). If you can't find it for some reason, make sure you're pointing at the `nuget` source and not the `TechSmith` source.